diff --git a/dependencies/rvdecoderdb/.gitmodules b/dependencies/rvdecoderdb/.gitmodules
index 040fd4c..9f5da4b 100644
--- a/dependencies/rvdecoderdb/.gitmodules
+++ b/dependencies/rvdecoderdb/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "rvdecoderdbtest/jvm/riscv-opcodes"]
 	path = rvdecoderdbtest/jvm/riscv-opcodes
-	url = git@github.com:riscv/riscv-opcodes.git
+	url = https://github.com/riscv/riscv-opcodes.git
diff --git a/backend/asic/yosys-sta/example/gcd.sdc b/backend/asic/yosys-sta/example/gcd.sdc
index 003ec6f..aed1b96 100644
--- a/backend/asic/yosys-sta/example/gcd.sdc
+++ b/backend/asic/yosys-sta/example/gcd.sdc
@@ -1,4 +1,4 @@
-set clk_port_name clk
+set clk_port_name clock
 set CLK_FREQ_MHZ 500
 if {[info exists env(CLK_FREQ_MHZ)]} {
   set CLK_FREQ_MHZ $::env(CLK_FREQ_MHZ)
